doc-content/enterprise-only/cases/case-management-case-file.con.adoc

```diff
@@ -22,3 +22,53 @@ You can specify any prefix. For example, if you specify the prefix `IT`, the fol
 `IT-0000000002`
 
 `IT-0000000003`
+
+You can also configure a variable as the business key, for example the following generates the `HR-TYPE1` prefix.
+
+[source]
+----
+{
+  case variable =>  type = "type1"
+  prefix = "HR-@{UPPER(type})"
+}
+----
+
+.Prefix HR-TYPE1
+
+[source]
+----
+{
+  HR-TYPE1-0000000001
+  HR-TYPE1-0000000002
+  HR-TYPE1-0000000003
+}
+----
+
+.Adding a different case variable, for example, ``type = 3type`
```

Comment:
No closing quote in rendered output.  Wondering if this is 100% clear for users, maybe rather than showing this as two separate lists (one for each value of type variable), wouldn't it make sense to just create one list (as typically these cases are started each with their own value): HR-type1-0000000001, HR-type2-0000000001, HR-type1-0000000002, HR-type2-0000000003, HR-type3-0000000001, HR-type3-0000000001, ...

Reply:
Updated layout.

Reply:
Done in Design doc. This topic removed for GSG.